src/app/pages/component-sidenav/_component-sidenav-theme.scss

```diff
@@ -43,6 +28,10 @@
         background: rgba(0, 0, 0, $nav-background-focus-opacity);
       }
     }
+
+    .mat-list-item {
+      color: rgba(mat-color($foreground, secondary-text), 0.6);
```

Comment:
The contrast ratio here is a bit low. Was there a reason for changing the text color?

Reply:
I thought the contrast was too high actually, was trying to go for something like this: https://standards.google/guidelines/google-material/components/
where the contrast is lower normally and increases on hover

Reply:
This ratio is only 5.66, which doesn't meet the W3C's AAA level:
https://contrast-ratio.com/#rgba%280%2C0%2C0%2C.6%29-on-%23fafafa

The ratio on that internal site is 6.05

Reply:
Changed to 0.65, now ratio is 6.86
https://contrast-ratio.com/#rgba%280%2C0%2C0%2C.65%29-on-%23fafafa